Frontend/src/components/Login.js

```diff
@@ -120,17 +123,35 @@ const Login = ({ setUser }) => {
     position: 'fixed'
   };
 
+  const dmImageStyle = {
```

Comment:
You could just reuse the `bgImage` code by having a ternary opertator where it says `url(${bgImage})`. Doesn't super matter tho

Reply:
Yanno fair. mind changing it? I'm studying for a 311 exam I have at 6 tonight

Reply:
Writing a paper rn but I can later